### PR TITLE
Add basecontroller to v8

### PR DESCRIPTION
**Pull request by Rochet2**
Adds the basecontroller to the v8.
This file is used by most (all?) other controllers in the v8.

@testmycode/api-team 

http://ohtu-jenkins.testmycode.io/job/v8basecontroller/4/

### CODE REVIEW

File: app/controllers/api/v8/base_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+require 'json'
+
+class Api::V7::BaseController < ApplicationController
+  clear_respond_to
+  respond_to :json
+
+  #  before_action :doorkeeper_authorize!
+  before_action :authenticate_user!
+
+  skip_authorization_check # we use doorkeeper in api, so let's skip cancancan
```
Comment (nygrenh) [inline app/controllers/api/v8/base_controller.rb]: Do we want this? Cancancan can most likely work with doorkeeper tokens.

File: app/controllers/api/v8/base_controller.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+require 'json'
+
+class Api::V7::BaseController < ApplicationController
+  clear_respond_to
+  respond_to :json
+
+  #  before_action :doorkeeper_authorize!
+  before_action :authenticate_user!
```
Comment (nygrenh) [inline app/controllers/api/v8/base_controller.rb]: Should we test this?

Comment (nygrenh) [inline app/controllers/api/v8/base_controller.rb]: You can write controller tests and make assertions on the `@current_user` instance variable. I think it's important this always works correctly :).

**Comment by Rochet2** [comment]
Jenkins succeeded http://ohtu-jenkins.testmycode.io/job/v8basecontroller/4/

**Pull request merged**